server/src/main/java/org/elasticsearch/rest/action/info/RestClusterInfoAction.java

```diff
@@ -50,7 +55,10 @@ public class RestClusterInfoAction extends BaseRestHandler {
         nodesStatsResponse -> nodesStatsResponse.getNodes()
             .stream()
             .map(NodeStats::getIngestStats)
-            .reduce(IngestStats.IDENTITY, IngestStats::merge)
+            .reduce(IngestStats.IDENTITY, IngestStats::merge),
+        //
+        THREAD_POOL.metricName(),
+        response -> response.getNodes().stream().map(NodeStats::getThreadPool).reduce(ThreadPoolStats.IDENTITY, ThreadPoolStats::merge)
```

Comment:
Nit, if I am not mistaken this `response` is the same as the `nodesStatsResponse` in the other metrics. I would prefer to have it consistent over all the metrics to show that they have the same input.